lib/lockfile.ml

```diff
@@ -261,6 +261,21 @@ module Depexts = struct
     List.concat all |> List.sort_uniq ~cmp:compare_elm
 end
 
+module Cli_args = struct
+  (** Field used to store the raw command line arguments passed to [lock].
+      It is set but not read and therefore is not part of the main lock file
+      type. *)
+
+  type _t = string list
```

Comment:
What is the reason to not name it `t`?

Reply:
The compiler rightfully complains it is unused otherwise. I still wanted to keep the type to clarify things in the code.